bot/exts/help_channels/_cog.py

```diff
@@ -20,6 +21,7 @@
 HELP_CHANNEL_TOPIC = """
 This is a Python help channel. You can claim your own help channel in the Python Help: Available category.
 """
+AVAILABLE_HELP_CHANNELS = """**Currently available help channel(s):** {available}"""
```

Comment:
Can we spread this across lines?
```suggestion
AVAILABLE_HELP_CHANNELS = """
**Currently available help channel(s):** {available}
"""
```

Reply:
https://github.com/python-discord/bot/pull/1396#discussion_r570897703

Reply:
It seems more readable, `"""` are multi line strings, so either change it to `"` single quotes or keep it as it was before
(my opinion)

Reply:
After looking at other cogs, it does seem that one liners should have single double quotes. An example can be seen [here](<https://github.com/python-discord/bot/blob/e5b073c5ee9c7c887a193ec05d813d259eca58ee/bot/exts/help_channels/_message.py#L30>).